src/main/resources/expat-site-default.xml

```diff
@@ -175,6 +175,7 @@
     </version-33>
     <version-34>
         io.hops.hopsworks.expat.migrations.featurestore.featureview.SetServingKeys
+        io.hops.hopsworks.expat.migrations.airflow.DagsMigration
```

Comment:
You need to make a PR on hopsworks-chef and cloud-chef to add this migration to expat-site.xml used during deployment/upgrades.